src/DynamoCLI/DynamoCLI.csproj

```diff
@@ -8,11 +8,12 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>DynamoCLI</RootNamespace>
     <AssemblyName>DynamoCLI</AssemblyName>
+    <SelfContained>false</SelfContained>
```

Comment:
just curious - what happens if this is true, the exe also includes all the binaries in the bin folder?

Reply:
SelfContained = true will copy all the dotnet6 runtime dlls